### ISSUE TITLE
Missing translations on order cancel modal

### ISSUE THREAD

**Issue by Akarshit** [reactioncommerce/reaction#2919]
### Expected behavior
The strings should be fetched from i18n files and then displayed

### Actual Behavior
![](https://i.imgur.com/THZmhzm.png)

### Steps to Reproduce the Behavior
1. Order something
2. Proceed, capture payment and then cancel the order.

### Versions
```
Node: 8.5.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.12.0
Reaction: 1.5.0
Reaction branch: marketplace
```

**Comment by aaronjudd** [comment]
More generally,  these missing labels are likely due to a) a corrupt *.json file, due to a bad merge, or b) missing files due to a bad merge.  The marketplace is not showing a lot of translation.  Tracking this in #2641

**Comment by spencern** [comment]
Closed by #2954
